go.mod

```diff
@@ -0,0 +1,59 @@
+module github.com/caicloud/cyclone
+
+go 1.12
```

Comment:
Upgrade your local Golang, this should be `go 1.13`

Reply:
Same as above
@iawia002 PTAL

Reply:
OK then